lib/sidekiq_unique_jobs/configuration.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module SidekiqUniqueJobs
+  # Shared class for dealing with gem configuration
+  #
+  # @author Mauro Berlanda <mauro.berlanda@gmail.com>
+  class Configuration
+    DEFAULT_LOCKS = {
```

Comment:
I'd be totally fine with moving this up the chain. Maybe even have this as a global constant under `SidekiqUniqueJobs` directly since it concerns more than just configuration. That way we could remove the other constant `LOCKS` in https://github.com/mhenrixon/sidekiq-unique-jobs/blob/master/lib/sidekiq_unique_jobs/options_with_fallback.rb#L12.

Reply:
Or possibly put it with the other constants to have all of them in the same place but I am kind of indifferent as long as everything gets wired up correctly.